app/controllers/teams_controller.rb

```diff
@@ -14,9 +14,16 @@ def index
   # GET /teams/1
   # GET /teams/1.json
   def show
+    raise ActiveRecord::RecordNotFound if @team.name.starts_with?("portus_global_team_")
     authorize @team
     @team_users = @team.team_users.enabled.page(params[:users_page]).per(10)
     @team_namespaces = @team.namespaces.page(params[:namespaces_page]).per(15)
+  rescue ActiveRecord::RecordNotFound
```

Comment:
It doesn't on development mode, but it does on staging/production :) Just try it with staging (so you don't have to do anything special in the config as in production), you'll get the fancy 404 page.